src/main/kotlin/com/zepben/evolve/services/network/translator/NetworkCimToProto.kt

```diff
@@ -1086,6 +1164,11 @@ fun toPb(cim: RatioTapChanger, pb: PBRatioTapChanger.Builder): PBRatioTapChanger
         toPb(cim, tcBuilder)
     }
 
+fun toPb(cim: ReactiveCapabilityCurve, pb: PBReactiveCapabilityCurve.Builder): PBReactiveCapabilityCurve.Builder =
```

Comment:
todo: check alphabetical order and correct package for all new calls, including extensions.

Reply:
moved a few around not sure if caught it all because there was a bunch of pre-existing out of order ones